examples/basic/android/build.gradle

```diff
@@ -44,4 +44,8 @@ ext {
     minSdkVersion = 19
     compileSdkVersion = 28
     targetSdkVersion = 28
+    // exoplayer_version - used in react-native-video and dice-shield-android
+    exoplayer_version = '2.9.6'
```

Comment:
Shouldn't this follow the camelcase convention? i.e. `exoplayerVersion`

Reply:
yes, you are right! I'll change it )